overlay_new.py

```diff
@@ -0,0 +1,197 @@
+from abc import ABC, abstractmethod
+from time import time
+
+from canvas import Canvas, Colour
+from data import Data
+from overlay import Overlay
+from textwrap import wrap
+from typing import Callable, Tuple
```

Comment:
Does typing enforce type checking?

Reply:
I wish lol. `typing` really only serves two purposes - It allows the reader to know what types things *should* have (kinda another form of documentation), and it also means that editors can know what types things are when they provide tab-completions. (e.g. if a parameter `s` was known to be a string, the editor can suggest `s.split()`, `s.lower()`, etc.)